lambdatimes/src/components/Content/Content.js

```diff
@@ -12,16 +13,25 @@ export default class Content extends Component {
     this.state = {
       selected: 'all',
       tabs: [],
-      cards: []
+      cards: [],
+      carosel:[]
```

Comment:
You don't really need this here I believe you removed it but must have forgotten to push afterwards.